src/commands/app/debug-proxy.ts

```diff
@@ -25,9 +25,17 @@ export default class DebugProxy extends Kommand {
       description: 'Port of the forwarding server',
       default: 9222
     }),
-    nonoAutoEnableDebugger: flags.boolean({
+    ttl: flags.string({
+      description: 'Kuzzle login TTL',
+      default: '1h',
+    }),
+    keepAuth: flags.boolean({
+      description: 'Keep the user authenticated',
+      default: false,
```

Comment:
I can't see where you're using those two new flags in the command code

Reply:
They are used by Kourou in the main class